src/structures/GuildChannel.js

```diff
@@ -258,13 +257,7 @@ class GuildChannel extends Channel {
    * @readonly
    */
   get members() {
-    const members = new Collection();
-    for (const member of this.guild.members.cache.values()) {
-      if (this.permissionsFor(member).has(Permissions.FLAGS.VIEW_CHANNEL, false)) {
-        members.set(member.id, member);
-      }
-    }
-    return members;
+    return this.guild.members.cache.filter(m => this.permissionsFor(m).has(Permissions.FLAGS.VIEW_CHANNEL));
```

Comment:
```suggestion
    return this.guild.members.cache.filter(m => this.permissionsFor(m).has(Permissions.FLAGS.VIEW_CHANNEL, false));
```

This doesn't do what it sounds like, it's *very* confusing. The reason this exists is because the api lets you overwrite the administrator permission on channel, however, it does not do anything, meaning it needs to be ignored. If you have admin from a role, it automatically sets all permissions to true so the administrator check is a noop for that too.

Reply:
Oh right, wasn't aware of that point